src/OnboardingSPA/pages/Steps/GetStarted/GetStartedExperience/index.js

```diff
@@ -21,7 +21,7 @@ const GetStartedExperience = () => {
 	const [ isLoaded, setisLoaded ] = useState( false );
 	const [ wpComfortLevel, setWpComfortLevel ] = useState( '0' );
 
-	const { setCurrentOnboardingData } = useDispatch( nfdOnboardingStore );
+	const { setCurrentOnboardingData, setSidebarActiveView } = useDispatch( nfdOnboardingStore );
```

Comment:
Can't we combine it with the useDispatch on line 33?

Reply:
Hmm, looks like these double useDispatches had already made it to trunk